### DISCUSSION TITLE
Why not working with any PC GTA Ginput?

### DISCUSSION THREAD

**Discussion by goodstuff626** [nefarius/DsHidMini#243] [Support]
I tried 'sixaxis' and all the different modes in dshidmini with Ginput for GTA III & Vice City V1.0 and the controls never work right like they should. What exactly do I have to do in order to use Dshidmini with Ginput?

**Comment by nefarius** [comment]
[You read and follow the docs](https://vigem.org/projects/DsHidMini/SCP-XInput-Bridge/#gta-San-Andreas).

**Reply by goodstuff626** [reply]
That video shows "DS3s pressure sensitive inputs in the PC Edition of GTA San Andreas Version 1.0". Is that needed with Ginput gta iii & vice city pc controls? I thought that is only for pressure sensitive controls for san andreas.

**Reply by nefarius** [reply]
If you'd have watched the video you would have seen that I show how to set up GInput, which ofc. applies to all games GInput supports 😉

**Reply by goodstuff626** [reply]
ok i will watch whole video. (just confused because i didn't see all 3 gta pc games in the title, so i assumed it was only for san andreas. my bad, thanks for the help.)

**Comment by goodstuff626** [comment]
I watched the whole video to see where i went wrong. I needed to add the XInput1_3.dll file from the x86 folder, not the x64. But even after doing that correctly, and setting dshidmini to SXS mode like the video says, it still does not make the controls correct. I also tried to edit the ginput.ini to see if anything in there needed to be changed to fix the problem. I tried PlayStationButtons=1 & EnableSixaxisHack=1, but neither setting helped fix the problem. I added these to the gtaIII directory - 
SilentPatchIII
GInputIII
SilentPatchDDraw
Followed instructions where to place everything, and it still is not working correctly. ps3 Start still changes camera angle, circle goes forward in menus, not x. What could I be doing wrong??
